.github/workflows/validate-migration.yaml

```diff
@@ -377,21 +337,21 @@ jobs:
           ./config "${CONFIG_ARGS[@]}"
           make --jobs="$(nproc)"
           
-          sudo make install
+          sudo make install 
           
           popd
           
           EOF
 
       - name: "Build P4 Fusion and run validate migration script"
         run: |
-          .github/workflows/validate-migration-scripts/build-p4-fusion-and-run-validate-migration.sh
+          timeout --verbose 1h .github/workflows/validate-migration-scripts/build-p4-fusion-and-run-validate-migration.sh
```

Comment:
Wrt the 1hr timeout, it seems to be taking 16min at max right now. So I recommend setting the timeout to be something more aggressive, like 25min or 30min.